.github/workflows/release-please.yml

```diff
@@ -17,21 +17,28 @@ jobs:
 
       # The logic below handles npm publication.  Each step is conditional on a
       # release having been created by someone merging the release PR.
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
+        with:
+          ref: refs/tags/${{ steps.release.outputs.tag_name }}
```

Comment:
optional nit: It would be helpful if you put some reason why we need to add this additionally instead of the hyperlink.

Reply:
I believe there is a race, since the default ${{ github.ref }} resolves to refs/heads/main.  If another change is merged between the release PR and the workflow run, this could check out the wrong thing.  Hence the change to an explicit ref built from the release tag name.

Reply:
Ack, thanks!